packages/@aws-cdk/cfnspec/CHANGELOG.md

```diff
@@ -1,3 +1,197 @@
+* AWS::MediaLive::Channel.StaticKeySettings (__added__)
+* AWS::MediaLive::Channel.TeletextDestinationSettings (__added__)
+* AWS::MediaLive::Channel.TemporalFilterSettings (__added__)
+* AWS::MediaLive::Channel.TimecodeConfig (__added__)
+* AWS::MediaLive::Channel.TtmlDestinationSettings (__added__)
+* AWS::MediaLive::Channel.UdpContainerSettings (__added__)
+* AWS::MediaLive::Channel.UdpGroupSettings (__added__)
+* AWS::MediaLive::Channel.UdpOutputSettings (__added__)
+* AWS::MediaLive::Channel.VideoCodecSettings (__added__)
+* AWS::MediaLive::Channel.VideoDescription (__added__)
+* AWS::MediaLive::Channel.WebvttDestinationSettings (__added__)
+* AWS::MediaLive::Input.InputDeviceRequest (__added__)
+* AWS::MediaLive::Input.InputDeviceSettings (__added__)
+* AWS::ApiGatewayV2::DomainName.DomainNameConfiguration SecurityPolicy (__added__)
+* AWS::ECS::Service.AwsVpcConfiguration SecurityGroups.DuplicatesAllowed (__deleted__)
+* AWS::ECS::Service.AwsVpcConfiguration Subnets.DuplicatesAllowed (__deleted__)
+* AWS::ECS::Service.AwsVpcConfiguration Subnets.Required (__changed__)
+  * Old: true
+  * New: false
+* AWS::ECS::Service.LoadBalancer ContainerName.Documentation (__changed__)
+  * Old: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-service-loadbalancers.html#cfn-ecs-service-loadbalancers-containername
+  * New: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-service-loadbalancer.html#cfn-ecs-service-loadbalancer-containername
+* AWS::ECS::Service.LoadBalancer ContainerPort.Documentation (__changed__)
+  * Old: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-service-loadbalancers.html#cfn-ecs-service-loadbalancers-containerport
+  * New: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-service-loadbalancer.html#cfn-ecs-service-loadbalancer-containerport
+* AWS::ECS::Service.LoadBalancer ContainerPort.Required (__changed__)
+  * Old: true
+  * New: false
+* AWS::ECS::Service.LoadBalancer LoadBalancerName.Documentation (__changed__)
+  * Old: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-service-loadbalancers.html#cfn-ecs-service-loadbalancers-loadbalancername
+  * New: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-service-loadbalancer.html#cfn-ecs-service-loadbalancer-loadbalancername
+* AWS::ECS::Service.LoadBalancer TargetGroupArn.Documentation (__changed__)
+  * Old: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-service-loadbalancers.html#cfn-ecs-service-loadbalancers-targetgrouparn
+  * New: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-service-loadbalancer.html#cfn-ecs-service-loadbalancer-targetgrouparn
+* AWS::ECS::Service.NetworkConfiguration AwsvpcConfiguration (__deleted__)
+* AWS::ECS::Service.NetworkConfiguration AwsVpcConfiguration (__added__)
```

Comment:
context: https://github.com/aws-cloudformation/cfn-python-lint/issues/1706

Reply:
As you mentioned in the issue, since the `ECS` resource is deployed in the CloudFormation Registry, its property types are case insensitive. Changing this now might result in resource replacement for existing users, we have seem something similar in https://github.com/aws/aws-cdk/pull/10201#discussion_r489034275.  Given this works now, and changing it might result in a breaking change, wouldn't be better to keep the previous casing?

Reply:
> Given this works now, and changing it might result in a breaking change, wouldn't be better to keep the previous casing?

Definitely better. Part of why I called out that context. The casing should not have been changed in the first place

Reply:
@PatMyron @NetaNir makes sense - so a patch to change `AwsvpcConfiguration` back to `AwsvpcConfiguration`